polaris-tokens/src/themes/base/color.ts

```diff
@@ -116,7 +116,8 @@ export type ColorBackgroundAlias =
   | 'nav-bg'
   | 'radio-button-bg-surface-disabled'
   | 'video-thumbnail-play-button-bg-fill-hover'
-  | 'video-thumbnail-play-button-bg-fill';
+  | 'video-thumbnail-play-button-bg-fill'
+  | 'badge-new-bg-fill';
```

Comment:
Not blocking but I noticed that this file appears to be in alpha order. If we want to preserve that I think this new values should be listed after `backdrop-bg`.